tests/rpc/v1/staking_test.go

```diff
@@ -0,0 +1,71 @@
+package v1
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	v1 "github.com/oasislabs/oasis-indexer/api/v1"
+	"github.com/oasislabs/oasis-indexer/tests"
+)
+
+var stakingEndHeight int64 = 8054649
+
+func makeTestAccounts() []v1.Account {
+	return []v1.Account{
+		{
+			Address:   "oasis1qp28vcurlx03y9exedzd9kfp7u2p0f0nvvv7h5wv",
+			Nonce:     1,
+			Available: 0,
+			Escrow:    0,
+			Debonding: 0,
+			Total:     0,
+		},
+		{
+			Address:   "oasis1qrj5x6twyjg0lxkz9kv0y9tyhzpxwq9u6v6sgje2",
+			Nonce:     0,
+			Available: 56900000000,
+			Escrow:    0,
+			Debonding: 0,
+			Total:     56900000000,
+		},
+	}
+}
+
+func TestListAccounts(t *testing.T) {
+	if _, ok := os.LookupEnv("OASIS_INDEXER_E2E"); !ok {
+		t.Skip("skipping test since e2e tests are not enabled")
+	}
+
+	tests.Init()
+
+	<-tests.After(stakingEndHeight)
+
+	var list v1.AccountList
+	err := tests.GetFrom("/consensus/accounts?minAvailable=1000000000000000000", &list)
+	require.Nil(t, err)
+	require.Equal(t, 1, len(list.Accounts))
+
+	// The big kahuna (Binance Staking).
+	require.Equal(t, "oasis1qpg2xuz46g53737343r20yxeddhlvc2ldqsjh70p", list.Accounts[0].Address)
```

Comment:
Hmm. This is the same init/seed file, yeah? But below we are "mocking" the accounts below?